app/src/main/java/com/stevesoltys/seedvault/ui/storage/StorageRootFetcher.kt

```diff
@@ -171,15 +179,10 @@ internal class StorageRootFetcher(private val context: Context, private val isRe
             availableBytes = null,
             isUsb = false,
             requiresNetwork = true,
-            enabled = !isInstalled || isRestore,
+            enabled = (!isInstalled && canInstall) || (isRestore && (!isInstalled && canInstall)),
```

Comment:
Can we maybe simplify this logic here? Would `isInstalled || canInstall` work as well? Just judging from the click listener here.